common/utils.c

```diff
@@ -428,13 +428,24 @@ TDNFYesOrNo(
 
     if(!pArgs->nAssumeYes && !pArgs->nAssumeNo)
     {
-        pr_crit("%s", pszQuestion);
-        while ((opt = getchar()) == '\n' || opt == '\r');
-        opt = tolower(opt);
-        if (opt != 'y' && opt != 'n')
-        {
-            dwError = ERROR_TDNF_INVALID_INPUT;
-            BAIL_ON_TDNF_ERROR(dwError);
+        while(1) {
+            pr_crit("%s", pszQuestion);
```

Comment:
do we need to do `fflush(stdout)` here?

Reply:
Good point - it may be needed because we don't send a new line, so it isn't flushed automatically. But I think it should be in `log_console()`. But I never saw any issue.